contracts/whitelisting/BasicTokenWhitelist.sol

```diff
@@ -2,6 +2,7 @@ pragma solidity ^0.4.24;
 
 import "@gnosis.pm/dx-contracts/contracts/base/TokenWhitelist.sol";
 
+
 contract BasicTokenWhitelist is TokenWhitelist {
     constructor () public {}
 }
```

Comment:
please remove CR

Reply:
Can you point out where you see carriage return (`\r`)? According to search there isn't one in the whole project. Files should also be normalized with LF endings in the repo thanks to `.gitattributes`

Reply:
<img width="1117" alt="screen shot 2019-01-22 at 15 12 34" src="https://user-images.githubusercontent.com/5994398/51537792-2d735800-1e58-11e9-86e0-98620009bdcb.png">

Reply:
Do you mean LF(`\n`) on line 5? Not carriage return?

Reply:
line 8 in the image above

Reply:
That is very strange because I simply don't see it.
Fixed(?) by editing here on github.